src/util/request.js

```diff
@@ -80,15 +80,18 @@ const sendRequest = async (url, headers, source, destination, method = enums.Htt
     return response
   } catch (error) {
     Logger.error(error)
-    throw ErrorHandler.Factory.createFSPIOPError(ErrorHandler.Enums.FSPIOPErrorCodes.DESTINATION_COMMUNICATION_ERROR, 'Failed to send HTTP request to host', error, source, [
+    const extenstionArray = [
       { key: 'url', value: url },
       { key: 'sourceFsp', value: source },
       { key: 'destinationFsp', value: destination },
       { key: 'method', value: method },
-      { key: 'request', value: JSON.stringify(requestOptions) },
-      { key: 'status', value: error.response.status },
-      { key: 'response', value: error.response.data }
-    ])
+      { key: 'request', value: JSON.stringify(requestOptions) }
```

Comment:
Should we be sending the requestOptions in the extensionList? 

1. It's going to be concatenated, which probably means its not going to be useful
2. It may have sensitive information that we should probably not include there (i.e. stuff from the switch the gateway)

Reply:
The proposed code change deals only with unhandled error, when error.response is undefined. I see your point. Still, as you know **truncateExtensions** is configurable, thus if set to false requestOptions is not going to be truncated. As for the other concern for sensitive information, I agree it might not be appropriate to keep it.